ops/systemd/export_workers.env

```diff
@@ -1,16 +1,23 @@
 ## General settings
-HOSTNAME=https://export.hotosm.org
+HOSTNAME=export-staging.hotosm.org
 DJANGO_SETTINGS_MODULE=core.settings.project
 
 ## SMTP Settings
-EMAIL_HOST_PASSWORD=<SECURE_SMTP_PASSWORD>
 EMAIL_HOST=smtp.mailgun.org
-EMAIL_HOST_USER=export@examplemail.com
+EMAIL_HOST_USER=export@example.org
+EMAIL_HOST_PASSWORD=<password>
+REPLY_TO_EMAIL=sysadmin@example.org
+
+##OAUTH 
+OSM_API_KEY=<Secret Key>
+OSM_API_SECRET=<OSM API Secret>
```

Comment:
If possible, please make this like the following.
```
OSM_OAUTH1_APP_CONSUMER_KEY=<SECURE_OSM_OAUTH1_APP_CONSUMER_KEY>
OSM_OAUTH1_APP_CONSUMER_SECRET=<SECURE_OSM_OAUTH1_APP_CONSUMER_SECRET>
```

I know the keys are long strings but it will remove any unambiguity from configuration